Sources/_MatchingEngine/Regex/AST/MatchingOptions.swift

```diff
@@ -110,6 +110,16 @@ extension AST {
   }
 }
 
+extension AST.MatchingOptionSequence {
```

Comment:
FWIW, when we pay off some technical debt, DSLTree will have fewer and fewer AST constructs in it.

In general, we have 3 things, the AST's representation, the API's representation, and the model type.

Literal/AST _lowers-to_ model
API _lowers-to_ model

It's possible in some situations for all 3 to be the same type, or for the model to also be made public in API.